client/src/api/api.ts

```diff
@@ -766,10 +766,10 @@ export type CourseEventDtoTypeEnum = typeof CourseEventDtoTypeEnum[keyof typeof
 export interface CourseScheduleItemDto {
     /**
      * 
-     * @type {object}
```

Comment:
Difference in types were found when I added tests. If this file is auto-generated, how to change types properly?